tests/std/tests/P0355R7_calendars_and_time_zones_time_zones/test.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
This test file has kind of a funny name, but I don't have any better suggestion 😝 

No change requested.